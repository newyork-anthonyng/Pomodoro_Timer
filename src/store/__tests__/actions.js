import {
    dispatchTogglePlayAction,
    dispatchPauseAction,
    dispatchResetAction,
    dispatchSetTimeAction,
    dispatchSetWorkIntervalAction,
    dispatchSetBreakIntervalAction
} from "../actions";

it("dispatchTogglePlayAction", () => {
    expect(dispatchTogglePlayAction()).toMatchSnapshot();
});

it("dispatchPauseAction", () => {
    expect(dispatchPauseAction()).toMatchSnapshot();
});

it("dispatchResetAction", () => {
    expect(dispatchResetAction()).toMatchSnapshot();
});

it("dispatchSetTimeAction", () => {
    expect(dispatchSetTimeAction(42)).toMatchSnapshot();
});

it("dispatchSetWorkIntervalAction", () => {
    expect(dispatchSetWorkIntervalAction()).toMatchSnapshot();
});

it("dispatchSetBreakIntervalAction", () => {
    expect(dispatchSetBreakIntervalAction()).toMatchSnapshot();
});