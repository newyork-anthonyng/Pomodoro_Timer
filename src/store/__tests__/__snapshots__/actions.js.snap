// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dispatchPauseAction 1`] = `
Object {
  "type": "pause_action",
}
`;

exports[`dispatchResetAction 1`] = `
Object {
  "type": "reset_action",
}
`;

exports[`dispatchSetBreakIntervalAction 1`] = `
Object {
  "type": "set_break_interval_action",
}
`;

exports[`dispatchSetTimeAction 1`] = `
Object {
  "data": 42,
  "type": "set_time_action",
}
`;

exports[`dispatchSetWorkIntervalAction 1`] = `
Object {
  "type": "set_work_interval_action",
}
`;

exports[`dispatchTogglePlayAction 1`] = `
Object {
  "type": "toggle_action",
}
`;

exports[`dispatchToggleWorkBreakIntervalAction 1`] = `
Object {
  "type": "toggle_work_break_interval_action",
}
`;
